src/vm/eventpipebuffermanager.cpp

```diff
@@ -883,6 +812,18 @@ void EventPipeBufferList::SetOwnedByThread(bool value)
     m_ownedByThread = value;
 }
 
+bool EventPipeBufferList::GetThreadEventWriteInProgress()
```

Comment:
nit: move `GetThreadEventWriteInProgress` and `SetThreadEventWriteInProgress` to the header file.

Reply:
done